File: Assignment No 3/Program2.c
//Write a program which accept number from user and print even factor of that number on screen.
//Input : 24
//Output : 1 2 4 6 8 12  
#include<stdio.h>

void DisplayFactor(int iNo)
{
	if(iNo <= 0)
	{
		iNo=-iNo;
	}
	int iCnt=0;

	for(iCnt=1;iCnt<iNo;iCnt++)
	{
		if(((iNo%iCnt)==0)&&((iCnt%2)==0))
		{
			printf("%d\t",iCnt);
		}
		
	}
}
int main()
{
	int iValue=0;
	printf("Enter number\n");

	scanf("%d",&iValue);
	DisplayFactor(iValue);
	return 0;
}

File: Assignment No 10/Program2.java
//Problem Statement : Write a Program which accept range from user and display all Even number in that range
//Input : 23 35
//Output :  24  26  28  30  32  34 
// Input : -10 2
//Output : -10  -8  -6  -4  -2 - 0  2 
// Input : 90 18
//Output :  Invalid range 
import java.io.*;

class Program2
{
	public static void RangeDisplayEven(int iStart,int iEnd)
	{
		if(iStart>iEnd)
		{
			System.out.println("Invalid Range");
		}
		for(;iStart<=iEnd;iStart++)
		{
			if(iStart%2==0)
			{
				System.out.print(iStart);
			System.out.print("\t");
			}
			
		}
	}
	public static void main(String args[]) throws Exception
	{
		int iValue1=0,iValue2=0;
		BufferedReader bobj= new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter Starting Point");
		iValue1=Integer.parseInt(bobj.readLine());


		System.out.println("Enter Ending Point");
		iValue2=Integer.parseInt(bobj.readLine());

		RangeDisplayEven(iValue1,iValue2);
	}
}